systemd-units/plymouth-poweroff.sh.in

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [ "$1" = "poweroff" ]; then
+	@PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
```

Comment:
@cosimoc that option is basically an hackish way to set the VT to KD_GRAPHICS instead of KD_TEXT when pymouth quits so that we do not see the white cursor on the black screen but instead we only see an uniform black screen.